pkg/scheduler/framework/plugins/interpodaffinity/scoring_test.go

```diff
@@ -773,9 +781,14 @@ func TestPreferredAffinity(t *testing.T) {
 			state := framework.NewCycleState()
 			p := plugintesting.SetupPluginWithInformers(ctx, t, New, &config.InterPodAffinityArgs{HardPodAffinityWeight: 1, IgnorePreferredTermsOfExistingPods: test.ignorePreferredTermsOfExistingPods}, cache.NewSnapshot(test.pods, test.nodes), namespaces)
 			status := p.(framework.PreScorePlugin).PreScore(ctx, state, test.pod, test.nodes)
+
 			if !status.IsSuccess() {
+				if status.Code() != test.wantStatus.Code() {
+					t.Errorf("InterPodAffinity#PreScore() returned unexpected status.Code got: %v, want: %v", status.Code(), test.wantStatus.Code())
+				}
+
 				if !strings.Contains(status.Message(), test.wantStatus.Message()) {
-					t.Errorf("unexpected error: %v", status)
+					t.Errorf("InterPodAffinity#PreScore() returned unexpected status.Message got: %v, want: %v", status.Message(), test.wantStatus.Message())
 				}
 			} else {
```

Comment:
reduce nest caused by `else`. (I know it's not your code though.)

```suggestion
			return
			}
```